example_configs/text2text/nmt-reversal-CC.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
I suggest to put all this "toy" configs (yours and RR, RT, TT) into the "toy-nmt-reversal" subfolder (or a similar name), because there are already too many of this configs and they are here only for the sake of sanity check, not doing anything very useful.